internal/infrastructure/mongo/mongodoc/asset_test.go

```diff
@@ -0,0 +1,147 @@
+package mongodoc
+
+import (
+	"testing"
+	"time"
+
+	"github.com/reearth/reearth-backend/pkg/asset"
+	"github.com/reearth/reearth-backend/pkg/id"
+	"github.com/stretchr/testify/assert"
+	"go.mongodb.org/mongo-driver/bson"
+)
+
+func TestNewAsset(t *testing.T) {
+	type args struct {
+		asset *asset.Asset
+	}
+
+	newAsset := asset.New().
+		NewID().
+		Team(id.NewTeamID()).
+		Name("test").
+		Size(10).
+		URL("test_url").
+		MustBuild()
+
+	tests := []struct {
+		name  string
+		args  args
+		want  *AssetDocument
+		want1 string
+	}{
+		{
+			name: "new asset",
+			args: args{
+				asset: newAsset,
+			},
+			want: &AssetDocument{
+				ID:          newAsset.ID().String(),
+				CreatedAt:   newAsset.CreatedAt(),
+				Team:        newAsset.Team().String(),
+				Name:        newAsset.Name(),
+				Size:        newAsset.Size(),
+				URL:         newAsset.URL(),
+				ContentType: newAsset.ContentType(),
+			},
+			want1: newAsset.ID().String(),
+		},
+	}
+	for _, tt := range tests {
+		tt := tt
+		t.Run(tt.name, func(t *testing.T) {
+			t.Parallel()
+			got, got1 := NewAsset(tt.args.asset)
+			assert.Equal(t, tt.want, got)
+			assert.Equal(t, tt.want1, got1)
+		})
+	}
+}
+
+func TestAssetDocument_Model(t *testing.T) {
+	aid, _ := id.AssetIDFrom("01f2r7kg1fvvffp0gmexgy5hxy")
+	tid, _ := id.TeamIDFrom("01f2r7kg1fvvffp0gmexgy5hxy")
+
+	newAsset := asset.New().
+		ID(aid).
+		CreatedAt(time.Time{}).
+		Team(tid).
+		Name("name").
+		Size(10).
+		URL("test").
+		ContentType("content type").
+		MustBuild()
+
+	tests := []struct {
+		name    string
+		target  *AssetDocument
+		want    *asset.Asset
+		wantErr bool
+	}{
+		{
+			name: "asset model",
+			target: &AssetDocument{
+				ID:          "01f2r7kg1fvvffp0gmexgy5hxy",
+				CreatedAt:   time.Time{},
+				Team:        "01f2r7kg1fvvffp0gmexgy5hxy",
+				Name:        "name",
+				Size:        10,
+				URL:         "test",
+				ContentType: "content type",
+			},
+			want:    newAsset,
```

Comment:
```suggestion
	now := time.Now()
	assetID := id.NewAssetID()
	teamID := id.NewTeamID()

	tests := []struct {
		name    string
		target  *AssetDocument
		want    *asset.Asset
		wantErr bool
	}{
		{
			name: "asset model",
			target: &AssetDocument{
				ID:          assetID.String(),
				CreatedAt:   now,
				Team:        teamID.String(),
				Name:        "name",
				Size:        10,
				URL:         "test",
				ContentType: "content type",
			},
			want: asset.New().
			      ID(assetID).
			      CreatedAt(now).
			      Team(teamID).
			      Name("name").
			      Size(10).
			      URL("test").
			      ContentType("content type").
			      MustBuild(),
```

It is easier to duplicate test cases and add various test cases if Asset is initialized for each test case.